api/v2/api.go

```diff
@@ -86,16 +88,18 @@ type (
 func NewAPI(
 	alerts provider.Alerts,
 	gf groupsFn,
-	sf getAlertStatusFn,
+	asf getAlertStatusFn,
+	gmf groupMutedFunc,
```

Comment:
How about this for consistent naming:

```suggestion
	asf AlertStatusFn,
	gmf groupMutedFn,
```

Reply:
I don't mind the types as these two are consistent:

```go
getAlertStatusFn func(prometheus_model.Fingerprint) types.AlertStatus
setAlertStatusFn func(prometheus_model.LabelSet)
```

It was more the variable names `asf` and `gmf` that I didn't like.